index.html

```diff
@@ -1956,39 +1956,9 @@ <h3>Securing Verifiable Credentials</h3>
 mutually exclusive.
         </p>
         <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be secured by at least
-one securing method.
+<a>Verifiable credentials</a> MUST be secured by at least one securing method.
         </p>
-        <p>
-Methods of securing <a>verifiable credentials</a> or
-<a>verifiable presentations</a> that embed a proof in the data model MUST use
-the <code>proof</code> <a>property</a>.
-        </p>
-        <p>
-Methods of securing <a>verifiable credentials</a> or <a>verifiable
-presentations</a> that use an external proof MAY use the <code>proof</code>
-<a>property</a>.
-        </p>
-        <dl>
-          <dt><var>proof</var></dt>
-          <dd>
-One or more cryptographic proofs that can be used to detect tampering and verify
-the authorship of a <a>verifiable credential</a> or <a>verifiable
-presentation</a>. The specific method used for an <a>embedded proof</a> MUST be
-identified using the <code>type</code> <a>property</a>.
-          </dd>
-        </dl>
-
-        <p>
-Because the method used for a mathematical proof varies by representation
-language and the technology used, the set of name-value pairs that is expected
-as the value of the <code>proof</code> <a>property</a> will vary accordingly.
-For example, if digital signatures are used for the proof mechanism, the
-<code>proof</code> <a>property</a> is expected to have name-value pairs that
-include a signature, a reference to the signing entity, and a representation of
-the signing date. The example below uses Ed25519 digital signatures.
```

Comment:
extraneous text, that is not longer relevant to the example below

Reply:
Why is it no longer relevant?